acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/DashboardPage.java

```diff
@@ -0,0 +1,40 @@
+package io.blueocean.ath.pages.blue;
+
+import io.blueocean.ath.BaseUrl;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class DashboardPage {
+    @Inject
+    WebDriver driver;
+
+    @Inject @BaseUrl
+    String base;
+
+    @Inject
+    public DashboardPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    public void open() {
+        driver.get(base + "/blue/");
+    }
+
+    public boolean isFavorite(String job) {
+        WebElement tr = driver.findElement(By.xpath("//tr[@data-name=\"" + job + "\"]"));
+        WebElement favorite = tr.findElement(By.cssSelector(".Checkbox.Favorite > label > input"));
+        return favorite.isSelected();
```

Comment:
It might be better to check if the card actually exists, rather than checking just the state of the star. But if you are just trying to illustrate something super simple, that's fine too... I'd be happy to beef up the favorites tests myself.